### PR TITLE
URL parameters for DSL

### PR DESCRIPTION
**Pull request by Kristjanmn**
Created URL variables for DSL files to read.
This ended up taking much more time than expected and trimmed down final version is extremely simple.
To introduce new variables to DSL:
1. Define variables in `application.yml` and `ApplicationProperties`
2. In `DslMappingHelper` point to new variables with `@Value("${application.dslParams.<myVariable>}")`
3. Call `replaceAll(myVariable, myValue)` on each node in `getDslSteps` function

### CODE REVIEW

**Comment by Kristjanmn** [comment]
@turnerrainer I believe this feature is ready now.

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -23,6 +23,9 @@ application:
     headers:
       Content-Type: "application/json"
   maxStepRecursions: 10
+  dslParameters:
+    DOMAIN_URL: "https://example.com"
+    PORT: 8080
```
Comment (turnerrainer) [inline src/main/resources/application.yml:28]: Remove this section as a whole from `application.yml` to `constants.ini`. This way, everything related to Ruuter's core setup remains separated from client-specific constants. And I mean _constants,_ not URLs specifically as these can be used throughout the DSL.

Comment (turnerrainer) [inline src/main/resources/application.yml:28]: Just to clarify - `DOMAIN_URL` and `PORT` are just sample keys, right? There can be any nr of them and all of them can be used in DSLs?

**Review by turnerrainer — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 1 above)
Comment (Kristjanmn) [reply src/main/resources/application.yml:28]: Correct.

**Review by turnerrainer — APPROVED** [review]

**Pull request merged**